.github/workflows/publish.yml

```diff
@@ -42,15 +42,15 @@ jobs:
           name: ${{ env.dists-artifact-name }}
           path: dist/
       - name: Publish to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.12.4
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I prefer keeping this pinned to a version and letting dependabot bump it when there's new changes, but I'm ok with as is it too.